test/scrabble_test.rb

```diff
@@ -6,8 +6,62 @@
 
 class ScrabbleTest < Minitest::Test
   def test_it_can_score_a_single_letter
-    skip
     assert_equal 1, Scrabble.new.score("a")
     assert_equal 4, Scrabble.new.score("f")
   end
+
+  def test_it_can_score_two_letters
+    assert_equal 2, Scrabble.new.score("as")
+    assert_equal 5, Scrabble.new.score("if")
+  end
+
+  def test_it_can_score_five_letters
+    assert_equal 8, Scrabble.new.score("tasty")
+    assert_equal 8, Scrabble.new.score("waste")
+  end
+
+  def test_returns_zero_score_for_nil_or_empty_input
+    assert_equal 0, Scrabble.new.score(nil)
+    assert_equal 0, Scrabble.new.score("")
+  end
+
+  def nil_or_empty_returns_false_for_word_input
+    refute nil_or_empty('r')
+    refute nil_or_empty('poop')
+  end
+
+  def nil_or_empty_returns_true_for_nil_or_empty_input
+    refute nil_or_empty(nil)
+    refute nil_or_empty('')
+  end
+
+  def upcase_and_split_returns_array_of_letters_for_single_letter
+    assert_equal 3, Scrabble.new.score("g")
+    assert_equal 1, Scrabble.new.score("l")
```

Comment:
Test name seems to indicate that this is a unit test. Would then expect that you test these methods directly. Otherwise these are basically still operating as integration tests. For example:

```ruby
def test_upcase_and_split_returns_an_array_of_letters_for_a_single_user
  scrabble = Scrabble.new

  assert_equal ['g'], scrabble.upcase_and_split('g')
  assert_equal ['l'], scrabble.upcase_and_split('l')
end
```